From 085c48345c0813e6b116a8af0afe921a1012b1f3 Mon Sep 17 00:00:00 2001
From: Jesse Hallett <jesse@sitr.us>
Date: Sat, 27 Apr 2024 20:14:13 -0700
Subject: [PATCH] patch launcher

---
 resources/1password.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/1password.desktop b/resources/1password.desktop
index 38efbd1..337b629 100644
--- a/resources/1password.desktop
+++ b/resources/1password.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=1Password
-Exec=/opt/1Password/1password %U
+Exec=/opt/1Password/1password %U --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations
 Terminal=false
 Type=Application
 Icon=1password
-- 
2.42.0

